src/Evolution/Systems/Ccz4/DerivChristoffel.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace Ccz4 {
+/// @{
+/*!
+ * \brief Computes the spatial derivative of the conformal spatial christoffel
+ * symbols of the second kind
+ *
+ * \details Computes the derivative as:
+ * \f{align}
+ *     \partial_k \tilde{\Gamma}^m{}_{ij} &=
+ *       \partial_k \tilde{\gamma}^{ml} (D_{ijl} + D_{jil} - D_{lij}) +
```

Comment:
this doesn't match the paper, which has `-2 D_k{}^{ml}`. This can actually matter because someone (like me) would interpret `\partial_k gamma^{ml}` as the numerical derivative. Your code will also need to be updated

Reply:
How do you feel about having the input argument be something like `field_d_up_times_two` for `2 D_k{}^{ij}` so that the argument already includes the factor of 2, along with a corresponding simple tag for the same value? This would be opposed to having an input variable and tag for `D_k{}^{ij}` without the factor of 2. It seems to be that most instances in the paper where `D_k{}^{ij}` is used, a factor of 2 is with it. (Open to better names than `FieldUpTimesTwo` for the tag, and likewise for the input variable)

Reply:
I'd stick to exactly what's in the paper for now and revisit later. I think this is trying to optimize things a bit early on :)